### ISSUE TITLE
`Info.plist` files ignored

### ISSUE THREAD

**Issue by eiditakahashi** [lyptt/struct#159]
## New Issue Checklist

- [ ] Updated struct to the latest version
- [x] I have read the [documentation](https://github.com/workshop/struct/wiki) to make sure my issue isn't covered already

## Struct version

`2.2.1`

## Spec version

`2.2.1`

## Which type of spec are you using?

- [x] project.yml
- [ ] project.json
- [ ] Specfile

## Context

`GoogleService-Info.plist` ignored 

## Reproduction steps

Run struct with `Info.plist` files in source folders

## Expected result

`Info.plist` files added to my project

## Actual result

No `Info.plist` files added to my project

---
Hi! I am trying struct in my project and got it running ok. I don't know if this is an issue, struct is ignoring all `Info.plist` files including `GoogleService-Info.plist` that is used with firebase in my project.

Removing this [line](https://github.com/lyptt/struct/blob/master/lib/spec/processor/target_sources.rb#L97) all `Info.plist` are added but you had a point when adding this line so I don't think I should do it... Do you have a suggestion to add at least `GoogleService-Info.plist` in the project?

**Comment by lyptt** [comment]
Hi @eiditakahashi, thanks for filing this issue. As far as I recall there’s no major issue which required I prevent adding Info.plist files other than the caveat that they shouldn’t be copied to the resources folder as this is taken care of by Xcode. 

I’ll investigate this in more detail tonight, should be an easy enough fix!

**Comment by lyptt** [comment]
Apologies for the delay on this one, I've gotten a couple of test faillures with the change to include non-Info.plist files. I should have a release ready soon!

**Comment by lyptt** [comment]
This is now fixed in 3.1.2.

**Comment by aradhyaalamuru** [comment]
Hi, just started using Struct. Seems awesome thank you.

I'm having the same issue -
'Info.plist' file inside './src' not getting copied. 

I tried with 'project.plist' file inside './src' and that is getting copied.

Details:
Struct version - 3.1.2
![screen shot 2018-08-05 at 9 21 37 pm](https://user-images.githubusercontent.com/17024041/43687607-93abd4aa-98f5-11e8-8384-ad49bade7f17.png)

project.yml 
```
---
version: 2.2.0
configurations:
  debug:
    profiles:
  release:
    profiles:
targets:
  ComplexApp:
    type: ":application"
    platform: ios
    sources: src
    references: 
    - sdkroot:UIKit.framework
    - sdkroot:sqlite3
```
Generated xcodeproj
![screen shot 2018-08-05 at 9 24 06 pm](https://user-images.githubusercontent.com/17024041/43687627-f243817a-98f5-11e8-97d8-62c6f1a6248b.png)

Anything I'm doing wrong?

Would love any help here.
Thank you!
